models/tmp/stg_stripe__charge_tmp.sql

```diff
@@ -13,6 +13,4 @@ select * from (
     )
 }}
 
-) as fields
-
-{{ livemode_predicate() }}
\ No newline at end of file
+) as fields
```

Comment:
Are the cte's necessary in this model anymore? I would imagine that with the removal of the livemode_predicate macro, we can just have the fivetran_utils.union_data macro and not need to have two ctes.

Reply:
Can you validate the above, if we do not need this then we should make the update in the other tmp models

Reply:
That's a good point, I don't think we'll need the CTEs. Locally, models runs fine without them. Removed them from the tmp models